### ISSUE TITLE
acs-engine scale reboots masters and rewrites network config

### ISSUE THREAD

**Issue by carlpett** [Azure/acs-engine#2662]
**Is this a request for help?**:
Yes

**Is this an ISSUE or FEATURE REQUEST?** (choose one):
ISSUE

**What version of acs-engine?**:
v0.15.1

**Orchestrator and version (e.g. Kubernetes, DC/OS, Swarm)**
Kubernetes 1.9.2

**What happened**:
We issued an `acs-scale` command to add more nodes to our running cluster, since we were getting low on free resources.

After the new nodes were created, the following unexpected things happened:
- All three masters were rebooted, simultaneously
- The masters got new IP addresses, which breaks at least etcd config (since it has the IP-addresses hard-coded from the initial cloud-init). Thus etcd refuses to start.

**What you expected to happen**:
- New nodes added
- No affect on existing nodes/masters

**How to reproduce it** (as minimally and precisely as possible):
From what we know so far, just `acs-engine scale`

**Anything else we need to know**:
This happened ~15 minutes ago, investigations and fixing still ongoing.

**Comment by CecileRobertMichon** [comment]
Hi @carlpett, what you're seeing is a side effect of PR #1966 (part of the 0.13.0 release) which was reverted by #2315 in v0.13.1... #1966 changed the default `firstConsecutiveStaticIP` which broke backwards compatibility for all previously built clusters. Reverting to the previous default of `"10.240.255.5"` fixed upgrade and scale for previously built clusters but unfortunately upgrading/scaling from clusters built with version 0.13.0 requires some manual tweaking. If building a new cluster is not an option for you, you might be able to get around this issue by switching all references to `"10.240.0.4"` in your apimodel and in your kubeconfig to `"10.240.255.5"`. 

FYI I just opened #2703, I agree that it would be useful to check the acs-engine version.

@JunSun17 might be able to help with this issue since he worked on reverting the change.
